# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/diffutils/hotfix-strcasecmp.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

checking whether strcasecmp works... configure: error: in '/usr/src/t2-src/src.diffutils.container.250413.112830.15352/diffutils-3.12':
configure: error: cannot run test program while cross compiling

diff --git a/m4/strcasecmp.m4 b/m4/strcasecmp.m4
index e40ee5d..eb4345d 100644
--- a/m4/strcasecmp.m4
+++ b/m4/strcasecmp.m4
@@ -1,5 +1,5 @@
 # strcasecmp.m4
-# serial 2
+# serial 3
 dnl Copyright (C) 2002-2025 Free Software Foundation, Inc.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
@@ -57,7 +57,7 @@ int main ()
           gl_cv_func_strcasecmp_works=no
         fi
        ],
-       [])
+       [:])
     ])
 ])

